kraken/kraken.json

```diff
@@ -2,119 +2,89 @@
   "version": 3,
   "timeout": "3s",
   "endpoints":[
-  {
-    "endpoint": "api/v1/health",
-    "extra_config": {
-      "auth/validator": {
-        "alg": "RS256",
-        "jwk_url": "http://keycloak:8080/realms/hms/protocol/openid-connect/certs",
```

Comment:
it was blocking me from requesting to backend, as we are validating token, even if it is missing we will return 401 if something is not like it should be